### PR TITLE
added new route client

### PR DESCRIPTION
**Pull request by Draaakos**

### CODE REVIEW

File: routes/client-routes.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    email: "hermano@gmail.com",
+    landPhone: 958458547,
+    mobilePhone: 958458547,
+    rank: 1,
+    comments: "new client"
+  }])
+};
+
+module.exports = function(app){
+  app.get('/clients', findAll);
```
Comment (aether7) [inline routes/client-routes.js]: It's needed to add a findById method, also we need these methods:

- create a new client
- update an existing client
- deleting a client

Obviously, these methods may be simulated for this purpose

File: routes/client-routes.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+function findAll(req, res){
+  res.json([{
+    id: 1,
+    firstName: "Brian",
+    lastName: "Soto",
+    secondlastName: "Farias",
+    rut: "17.485.265-8",
+    email: "hermano@gmail.com",
+    landPhone: 958458547,
```
Comment (worker-8) [inline routes/client-routes.js]: We are going to accept only one phone number (cell phone or line) or they must go both

File: routes/client-routes.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+function findAll(req, res){
+  res.json([{
+    id: 1,
+    firstName: "Brian",
+    lastName: "Soto",
+    secondlastName: "Farias",
+    rut: "17.485.265-8",
```
Comment (worker-8) [inline routes/client-routes.js]: With Dots and Dashboards

File: routes/client-routes.js (hunk 1 above)
Comment (Draaakos) [inline routes/client-routes.js]: okay i'll advance tomorrow in this problem, thanks you!.

File: routes/client-routes.js (hunk 2 above)
Comment (Draaakos) [inline routes/client-routes.js]: okay, i'll change this data, thanks you!

Comment (Mentalchaos) [inline routes/client-routes.js]: I think it is better to use both phone numbers cuz some people don't have a landline phone  (for example I don't have one) and maybe some people do not have a cellphone.
In that way you can choose which one do you want to use (or you can use both phone numbers if you want too)

File: routes/client-routes.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    email: "hermano@gmail.com",
+    landPhone: 958458547,
```
Comment (Mentalchaos) [inline routes/client-routes.js]: function

You are missing the fields **direccion** , **region** , **comuna** and **tipo cliente**

File: routes/client-routes.js (hunk 3 above)
Comment (Mentalchaos) [inline routes/client-routes.js]: Are you talking about the input **rut**?
I think it is better to have that field without dash and dots  in the database (for query optimizations and fastest searchs , etc)
but in the frontend we can use that kind of format for **rut** and if people wants to write it like this **12345678-9** we just could reorder the dash & dots.

**Pull request merged**